src/components/StartQuizPage/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Button, Form, Input } from 'antd';
+
+import  './index.css';
+
+const QuizStart = () => {
+    return (
+        <div className="quizStart" >
+            <Form id='form'>
+                <Form.Item
+                    name="username"
+                    rules={[{
+                        required: true,
+                        message: 'Please enter your username!',
+                        pattern: '^[a-zA-Z]+( [a-zA-Z]+)?$'
```

Comment:
Even if you remove everything after the first plus sign, the regex is the same right? Also, I think underscore and dot should be allowed. Maybe this `^[a-zA-Z._]+$`?